bitnami/keycloak/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 name: keycloak
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/keycloak
-version: 24.2.2
```

Comment:
From the chart perspective, this is not a major change. 
The change takes place internally (with the container version change) without any non-backward compatible change, except older versions of the container are supported. However, that is not a problem since our charts do not officially support running with older images.

From a chart user perspective, the change would be seamless and no action would be required.

Reply:
Please bump a patch version instead (24.2.3)

Reply:
Done! Thank you.